// KT-9733 No error shown for 2 "main" functions in the same file

fun main(args: Array<String>) {}
fun main(args: Array<String>) {}